examples/Nexrad_S3_Demo.py

```diff
@@ -0,0 +1,117 @@
+"""
+Plotting AWS-hosted NEXRAD Level 2 Data
+=======================================================
+
+Access NEXRAD radar data via Amazon Web Services and plot with MetPy
+
+Accessing data remotely is a powerful tool for big data, such as NEXRAD radar data.
+By accessing it in the cloud, you can save time and space from downloading the data locally.
+
+"""
+
+from io import BytesIO
+
+import boto3
+import botocore
+from botocore.client import Config
+import matplotlib.pyplot as plt
+from metpy.io import Level2File
+from metpy.plots import add_timestamp, ctables
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
+
+######################################################################
+# Access the data in the AWS cloud. Note that you'll need an AWS account (free at
+# https://aws.amazon.com/) and then set up security keys under the "My Security Credentials"
```

Comment:
It's a little odd to me to mention needing to create and account and set up credentials, and then use unsigned access below. Maybe just cut out the part about credentials?